src/main/kotlin/AppModule.kt

```diff
@@ -47,49 +50,65 @@ fun Application.module() {
         }
         route("/") {
             get {
-                call.respond(ThymeleafContent("index", mapOf("obj" to "")))
+                call.respond(ThymeleafContent("index", mapOf("obj" to ""), locale = Locale("vi")))
```

Comment:
this shouldn't default to Vietnamese

Reply:
Oh yes I've got used to that for a while. Now changing back